templates/solidity-frameworks/foundry/packages/nextjs/scaffold.config.ts.args.mjs

```diff
@@ -1 +1,3 @@
-export const chainName = 'foundry'
\ No newline at end of file
+export const configOverrides = {
+  targetNetworks: ["$$chains.foundry$$"]
```

Comment:
Oh damn, this is how people need to use it, right?

Can you explain in which cases this is needed? Sorry, I'm sure we've talked about this a bunch of times.

Reply:
> Oh damn, this is how people need to use it, right?

Yep

> Can you explain in which cases this is needed?

We need to use it when we want the result (variable or expression) to be used by js runtime. In our case we want the result: `targetNetworks: [chains.foundry]` where `chains.foundry` is a variable used in runtime.

We can't use `targetNetworks: ["chains.foundry"]` or `targetNetworks: "[chains.foundry]"` in template because result will be the same (`targetNetworks: ["chains.foundry"]` or `targetNetworks: "[chains.foundry]"` )

We added info about it [here](https://github.com/scaffold-eth/create-eth/blob/beta/contributors/TEMPLATING.md#rules-for-template-args)

Reply:
Ok, thanks! I didn't look at the PR closely and had the impression that we only needed to use those "wildcards" on the template file, not the template `args`.

It's ugly as hell, but I guess we explored a bunch of options and this was the best.